tests/check_files/test_check_files/test_system_check_files.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import pytest
+from datetime import datetime
+from deepdiff import DeepDiff
+
+OUTPUT_FILE = f"system_checkfiles_{datetime.now().timestamp()}.json"
+
+
+@pytest.fixture
+def get_first_file(request):
+    """Allows to use the --before-file in order to pass the file before an installation, update, or uninstallation.
```

Comment:
```suggestion
    """Allow to use the --before-file parameter in order to pass the file before making any changes to the environment.
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/2246/commits/1126de5ecc83eecbbc8d9271ae065b3b8420a94c